pkg/registry/navigator/cassandracluster/strategy.go

```diff
@@ -89,8 +89,10 @@ func (cassandraClusterStrategy) Canonicalize(obj runtime.Object) {
 }
 
 func (cassandraClusterStrategy) ValidateUpdate(ctx genericapirequest.Context, obj, old runtime.Object) field.ErrorList {
-	cass := obj.(*navigator.CassandraCluster)
-	return validation.ValidateCassandraCluster(cass)
+	newCassandraCluster := obj.(*navigator.CassandraCluster)
+	oldCassandraCluster := old.(*navigator.CassandraCluster)
+
+	return validation.ValidateCassandraClusterUpdate(oldCassandraCluster, newCassandraCluster)
```

Comment:
:grey_question: I'm still unclear what happens when I do `kubectl replace --file my/cassandra/manifest.yaml` Will that bypass this update check? Or only when I do `kubectl replace --force` ?

Reply:
It doesn't seem to bypass it:
```
 g/s/g/j/navigator {cassandra-multiple-nodepools} $ k replace -f cassandra.yaml                             
The CassandraCluster "mydb" is invalid: spec.nodePools[0].datacenter: Forbidden: cannot modify datacenter   
```

with --force, it gets accepted though:
```
 g/s/g/j/navigator {cassandra-multiple-nodepools} $ k replace -f cassandra.yaml --force                                                                                                                                                                                                                                  407ms 
cassandracluster "mydb" deleted
cassandracluster "mydb" replaced
```